content/reference/sql/version-control/merges.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Merges
+---
+
+# Merging branches
+
+To merge a branch into your current branch, use the [`DOLT_MERGE()`
+stored procedure](dolt-sql-procedures.md#dolt_merge):
+
+```sql
+call dolt_merge('feature-branch');
+```
+
+This function returns whether the merge was a fast-forward or not and whether or not there are conflicts generated by the merge.
```

Comment:
the conflicts column is also `1` when there are constraint violations:

```suggestion
This function returns whether the merge was a fast-forward or not and whether or not there are conflicts/constraint violations generated by the merge.
```

Reply:
You could clarify that here, or in a separate doc?

Reply:
We could also add a new column called `constraint_violations`

Reply:
I think we should.

Reply:
You can fix this later.